src/utils.js

```diff
@@ -46,11 +75,11 @@ export default class Utils {
    * @param {String} id - The data id of the task
    * @param {Boolean} val - The value of completed
    */
-  change(id, val) {
+  static change(id, val) {
     const tasks = JSON.parse(localStorage.getItem('tasks'));
     const filtered = tasks.filter((item) => item.index !== parseInt(id, 10));
     const task = tasks.find((item) => item.index === parseInt(id, 10));
-    task.completed = val;
+    typeof val === 'boolean' ? task.completed = val : task.description = val;
```

Comment:
- [ ] I think that if you re-write this in the same manner you did on lines 153 to 157 you won't have to disable the following eslint rule: `/* eslint-disable no-unused-expressions */`

>  if (typeof val === 'string') {
      remTasks = tasks.filter((item) => item.index !== parseInt(val, 10));
    } else {
      remTasks = tasks.filter((item) => item.completed === val);
    }